editors/code/tests/runTests.ts

```diff
@@ -0,0 +1,46 @@
+import * as path from 'path';
+import * as fs from 'fs';
+
+import { runTests } from 'vscode-test';
+
+async function main() {
+    try {
```

Comment:
I'd suggest to avoid big `try/catch` blocks and just
```ts
async function main {
    // ... no try/catch here
}

main().catch(err => {
    console.errro(err);
    process.exit(1);
});
```